### PR TITLE
div centered

### PR DESCRIPTION
**Pull request by perlus3**

### CODE REVIEW

**Review by NerdbordMentor — APPROVED** [review]
## Code Review Report

**Task goal:** Center horizontally, and vertically our `.box` in `.arena`.

**Requirements:**

1. Styles should be written in `styles.css` file.
2. `div` with `.box` class should be in the middle of `div` with class `.arena`.

**Review:**

The changes made to the `styles.css` file are in line with the task requirements. 

- The `.box` class has been updated to include `margin: auto;`. This will center the box within its container when used in conjunction with `display: flex;` on the parent container.
- The `.arena` class now includes `display: flex;`. This will make all direct children of elements with this class become flex items, and they will be laid out along a line according to the flexbox model. This is necessary for the `margin: auto;` in the `.box` class to center the box within the arena.

In conclusion, the code changes cover the task requirements. The `div` with `.box` class should now be centered horizontally and vertically within the `div` with `.arena` class.
